src/Forum/ForumServiceProvider.php

```diff
@@ -131,11 +131,11 @@ public function register()
         });
     }
 
-    public function boot(Container $container, Dispatcher $events, Factory $view)
+    public function boot(Container $container, Dispatcher $events, ViewFactory $views)
     {
-        $this->loadViewsFrom(__DIR__.'/../../views', 'flarum.forum');
+        $views->addNamespace('flarum.forum', __DIR__.'/../../views');
 
-        $view->share([
+        $views->share([
```

Comment:
aren't these changes unnecessary now that we're not implementing override ?

Reply:
Yes, but wouldn't this make it easier for extensions to override Flarum's views, and be more consistent with how developers would expect views to be added?

Reply:
how so ? it does the same thing as before, the difference is just that before it added the namespace after the view factory was resolved, only reason I'm mentioning this is because I don't know why that was done that way, perhaps there is a good reason to avoid resolving the view factory too early. That's why I generally like to avoid unnecessary changes. Otherwise I really don't know to be honest. Just a rather stay safe since it's not a necessary change type of thing.